client/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Container, Grid, InputLabel, Paper, TextField, Typography } from '@material-ui/core';
+import { useTheme } from '@material-ui/core/styles';
+import useStyles from './useStyles';
+import clsx from 'clsx';
+import { CustomButton } from '../../components/Button/CustomButton';
+import { CustomInput } from '../../components/Input/CustomInput';
+
+const FormElement = ({ children }: { children: JSX.Element[] | JSX.Element }): JSX.Element => {
+  return (
+    <Box display="flex" flexDirection="column" marginBottom={3}>
+      {children}
+    </Box>
+  );
+};
+
+const Home = (): JSX.Element => {
+  const theme = useTheme();
+  const { root, leftLogoContainer, leftLogoText, leftBodyContainer, slogan, label, dateInOff, right, background } =
+    useStyles();
+  return (
+    <>
+      <Grid container className={root}>
+        <Grid item xs={12} md={6} style={{ padding: theme.spacing(4) }}>
```

Comment:
Can use system props here for padding ([Ref](https://mui.com/components/grid/#system-props))

Reply:
Sorry! My bad, my reference was for v5 and your project is on v4. Looks like it's a new implementation.

Inline styling is still discouraged so it's great that you were able to pass the padding as a prop in your change. Giving the grid a class in `useStyles` to set the padding may be easier, but I may have given the impression that you're required to pass it in as a prop.